### DISCUSSION TITLE
Approx timeframe for QField Cloud release?

### DISCUSSION THREAD

**Discussion by timbo-barnes** [opengisch/QField#3667] [Q&A]
I realize this is a tricky question which developers hate to answer :P but, do we have an indication of when QFieldCloud will come out of beta and be released for use? 
Are there specific bottle necks for which funding could be contributed to in order to speed up release?

**Comment by suricactus** [comment]
Very good guess that developers hate answering such questions. :) But I can assure you we are implementing the final features before going out of beta, so the time is coming much sooner than previous "soon"-s :) .

**Reply by timbo-barnes** [reply]
Great stuff, thank you :)
